modules/extensions-xml-support/src/test/resources/module-documentation/module-documentation.xml

```diff
@@ -12,10 +12,12 @@
            http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd">
 
     <property name="aPropertyWithDoc" type="string" doc:description="Documentation for the property"/>
+    <property name="aHiddenPropertyWithDoc" type="string" password="true" doc:description="Documentation for the hidden property"/>
 
     <operation name="operation-with-doc" doc:description="Documentation for the operation">
         <parameters>
             <parameter name="paramWithDoc" type="string" doc:description="Documentation for the parameter" />
+            <parameter name="hiddenParamWithDoc" type="string" password="true" doc:description="Documentation for the hidden parameter" />
```

Comment:
There might be other types of elements in the future, but TBH I'd say the missing attributes today are ralated to display things, such as: friendly name, tab name, order, and so on.

About complex types of parameters, like _email parameter_ are really ahead in the road and I'd not pay attention at this time.